src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -29,10 +25,11 @@ internal abstract partial class MemberSemanticModel : CSharpSemanticModel
     {
         private readonly Symbol _memberSymbol;
         private readonly CSharpSyntaxNode _root;
-        private readonly DiagnosticBag _ignoredDiagnostics = new DiagnosticBag();
-        private readonly ReaderWriterLockSlim _nodeMapLock = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
+        private readonly DiagnosticBag _ignoredDiagnostics = new();
+        private readonly ReaderWriterLockSlim _nodeMapLock = new(LockRecursionPolicy.NoRecursion);
         // The bound nodes associated with a syntax node, from highest in the tree to lowest.
-        private readonly Dictionary<SyntaxNode, ImmutableArray<BoundNode>> _guardedNodeMap = new Dictionary<SyntaxNode, ImmutableArray<BoundNode>>();
+        private readonly Dictionary<SyntaxNode, ImmutableArray<BoundNode>> _guardedBoundNodeMap = new();
```

Comment:
>new(); [](start = 98, length = 6)

Same comment here and below

Reply:
BTW, this changes like that do not follow a general guideline we have to not bundle unrelated refactorings and style changes with product changes in a single PR.

---
In reply to: [526947477](https://github.com/dotnet/roslyn/pull/49453#discussion_r526947477) [](ancestors = 526947477)